includes/general.php

```diff
@@ -1052,7 +1052,16 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		if ( ! empty( $tags['template'] ) ) {
+			$return  = '';
+			// @todo New Pods query based on relationship field.
```

Comment:
:warning: This comment is 47% valid code; is this commented out code?

<sub>hash: 5d702fc7856cce018653f3da4bd128dc</sub>